packages/flutter/lib/src/material/curves.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/animation.dart';
+
+// The easing curves of the Material Library
```

Comment:
This is a comment instead of being part of the documentation, is that intentional or should it be `///`?

Reply:
This is intentional